mirage/config.js

```diff
@@ -136,6 +136,15 @@ export default function() {
   });
 
   /**
+   * Donation goals
+   */
+
+  this.get('/donation-goals', { coalesce: true });
+  this.get('/donation-goals/:id');
+  this.patch('/donation-goals/:id');
+  this.post('/donation-goals');
```

Comment:
I really don't think we need to comment on each and every mirage config shorthand. A comment like

```Javascript
// POST /categories
this.post('/categories');
```

really just tells us what the code is already telling us.